ledger/api-server-damlonx/reference-v2/src/main/scala/com/daml/ledger/api/server/damlonx/reference/v2/ReferenceServer.scala

```diff
@@ -61,13 +62,18 @@ object ReferenceServer extends App {
 
   val participantLoggerFactory = NamedLoggerFactory.forParticipant(participantId)
   val participantF: Future[(AutoCloseable, StandaloneIndexServer#SandboxState)] = for {
-    indexerServer <- StandaloneIndexerServer(readService, config, participantLoggerFactory)
+    indexerServer <- StandaloneIndexerServer(
+      readService,
+      config,
+      participantLoggerFactory,
+      SharedMetricRegistries.getOrCreate(s"indexer-$participantId"))
```

Comment:
Perhaps pull the registry out (and `SharedMetricRegistries.getOrCreate(s"ledger-api-server-$participantId")` too), as they're used twice.

Reply:
`participantId` refers to two different IDs.

Reply:
Good point. Can we avoid the shadowing by renaming the other one to `extraParticipantId` or something?

Reply:
Addressed by 1227da2.